change/@ni-nimble-components-4d50372b-5cd2-4e13-9053-d023da4a2ebe.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If this doesn't resolve #1151 and you don't plan to do so via follow-up PRs, could you make sure the description there is updated to capture what's done and what you know is left to do?